### ISSUE TITLE
[Screen Reader-Project Documantation]: State is not defined for the links in the left pane which can Expand/Collapse. 

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2799]
**User Experience:**
User who depends on screen reader may get misguided about the links if state is not defined for the links which can Expand/Collapse.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.
Screen Reader : Narrator.

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/projects
2. Navigate to left panel controls and verify that state is defined for the links which can Expand/Collapse.

**Actual Result:-**
State is not defined for the links in the left pane which can Expand/Collapse. 

**Expected Result:-**
State should be defined for the links in the left pane as Expand/Collapse.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Refer Attachment**
[MAS4.1.2_Make Code_Project_State of the links.zip](https://github.com/Microsoft/pxt/files/1248575/MAS4.1.2_Make.Code_Project_State.of.the.links.zip)

![mas4 1 2_makecode_state is not defined for the link](https://user-images.githubusercontent.com/30434417/29668177-de8e98cc-88fc-11e7-9040-ed8f6536408d.png)

**Comment by veler** [comment]
Issue fixed couple of weeks ago but still not deployed.

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/projects and issue is fixed now. Hence closing the bug
